assets/yaml/us/ast_logo.yml

```diff
@@ -0,0 +1,15 @@
+:config:
+  segments:
+    seg15: 0xD25100
+  header:
+    - '#include "assets/ast_logo.h"'
+    - // clang-format off
```

Comment:
correct me if i'm wrong but i don't think the clang-format we have set goes through yaml files.